regression/goto-instrument/show-call-sequences4/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+main.c
+--show-call-sequences
+activate-multi-line-match
+main -> moo\nmain -> foo
+foo -> moo\nfoo -> boo
+EXIT=0
+SIGNAL=0
+
```

Comment:
This is supposed to show case that function calls per function are grouped together - hence the multiline. Of course, unexpected cases wont' hurt. But that's the feature that the test is supposed to verify